app/components/selections/NewOrganizationMultiSelectInput/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useMemo } from 'react';
+
+import {
+    SearchMultiSelectInput,
+    SearchMultiSelectInputProps,
+} from '@the-deep/deep-ui';
+import { useQuery, gql } from '@apollo/client';
+
+import {
+    OrganizationOptionsQuery,
+    OrganizationOptionsQueryVariables,
+} from '#generated/types';
+import useDebouncedValue from '#hooks/useDebouncedValue';
+
+const ORGANIZATIONS = gql`
+    query OrganizationOptions($search: String) {
+        organizations(search: $search) {
+            results {
+                id
+                title
+                mergedAs {
+                    id
+                    title
+                }
+            }
+            totalCount
+        }
+    }
+`;
+
+export type BasicOrganization = NonNullable<NonNullable<NonNullable<OrganizationOptionsQuery['organizations']>['results']>[number]>;
+
+type Def = { containerClassName?: string };
+type OrganizationMultiSelectInputProps<K extends string> = SearchMultiSelectInputProps<
+    string,
+    K,
+    BasicOrganization,
+    Def,
+    'onSearchValueChange' | 'searchOptions' | 'optionsPending' | 'keySelector' | 'labelSelector' | 'totalOptionsCount' | 'onShowDropdownChange'
+>;
+
+const keySelector = (d: BasicOrganization) => d.id;
+const labelSelector = (d: BasicOrganization) => d.title;
```

Comment:
I am always confused about this. Should the title be `title` or `mergeAs.title`?

Reply:
This is fine, as new searches won't return merged values

Reply:
Noh!